test/functional/lua/cursor_pending_operator_spec.lua

```diff
@@ -0,0 +1,1428 @@
+--- Make sure the basic functionality of cursor-text-object works.
```

Comment:
these tests should live in `test/functional/editor` . the lua/ subdir is for the Lua "stdlib"